pkg/imagestore/imagestore.go

```diff
@@ -147,9 +151,33 @@ func validateVersions(versions []map[string]string) error {
 	return nil
 }
 
-func (s *rhcosStore) downloadURLToFile(url string, path string) error {
-	resp, err := s.httpClient.Get(url)
+func (s *rhcosStore) getHttpResponse(url string) (*http.Response, error) {
```

Comment:
I find `getHttpResponse` to be an odd name for something that makes an HTTP request, I think if we call it `doHttpRequest` or `performHTTPRequest` or something, the response return value is already implied